www/css/coolhole.css

```diff
@@ -398,17 +398,107 @@
 }
 
 /* CoolPoints User Styling */
-#coinWrapper {
```

Comment:
nit: While the UI looks _much_ better (thanks for that I have no eye for design), I think there's a few things here that hurt us or could be improved

![image](https://github.com/user-attachments/assets/3f07a034-3791-4e96-bff7-43840143756b)
1. I think we could use some padding here on all the inner content in the modal. The title and the tabs certainly don't need it
2. Might be personal preference but I think I like how the title looks centered
![image](https://github.com/user-attachments/assets/8143cd60-2cd1-44b9-8201-58a6609c11e1)
3. I think the white close icon is good but it's not consistent with the rest of the modals. If we want to change this we might wanna consider finding the style responsible in the theme

![image](https://github.com/user-attachments/assets/895ce27c-4ec5-462e-a558-dd9b810f540d)
Casino real busted (not that it wasn't before)

![image](https://github.com/user-attachments/assets/3a41f12d-dfc3-4c11-adaa-eaaba1b393f4)
Status of actions could use some help if you're willing

Reply:
You read my mind, I'm just fighting all of the Bootstrap 4/ Finalboss, `.row` class abuse, I was already pretty tired last night and tracking down why the modal was padding but the dividers weren't was driving me insane. I do plan to finish centering and revising all this later today.